composer.json

```diff
@@ -11,13 +11,13 @@
     }
   ],
   "require": {
-    "symfony/framework-bundle": "~2.1",
+    "symfony/framework-bundle": "~2.1|~3.0",
     "willdurand/geocoder": "~3.1",
```

Comment:
mmh nevermind, I thought we add a dependency to a Symfony component, but that's not the case..